build.js

```diff
@@ -16,8 +16,8 @@ const redirectsFile = path.join(outputFolder, '_redirects');
 const redirects = [];
 
 snx.networks
-	.filter(network => network !== 'local')
-	.forEach(network => {
+	.filter((network) => network !== 'local')
```

Comment:
@andytcf can you check your ES Lint settings? It shouldn't be adding these changes unless something isn't configured properly

Reply:
@justinjmoses fixed it